dev-docs/modules/topicsFpdModule.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page_v2
+page_type: module
+title: Module - topicsFpdModule
+description: how to get topics data using Topics API
+module_code : topicsFpdModule
+display_name : Topics FPD Module
```

Comment:
Is this conceptually not an RTD module? I haven't looked an the pbjs implementation, but from a functional point of view it provides real time data?

```
module_type: rtd
```

Reply:
It is related to First Party data. Don't know what will be the module_type in this case.

Reply:
@patmmccann Can you please confirm on this ?

Reply:
@patmmccann  can you please confirm on this ? Do we need to add module_type: rtd in this case.
@muuki88 already has given confirmation that PR can be merged.

Reply:
Good to merge for me